@types declarations seems to be missing

When building with latest read-config-file, it seems to be missing a corresponding `.d.ts` file
```
packages/app-builder-lib/src/util/config.ts:6:100 - error TS7016: Could not find a declaration file for module 'read-config-file'. '/Users/michaelmaietta/Development/electron-builder/node_modules/.pnpm/read-config-file@6.3.1/node_modules/read-config-file/out/main.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/read-config-file` if it exists or add a new declaration (.d.ts) file containing `declare module 'read-config-file';`

6 import { getConfig as _getConfig, loadParentConfig, orNullIfFileNotExist, ReadConfigRequest } from "read-config-file"
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -12,7 +12,8 @@
     "allowSyntheticDefaultImports": true,
     "newLine": "lf",
     "noEmitOnError": true,
-    "outDir": "out"
+    "outDir": "out",
+    "declaration": true
   },
   "include": [
     "src/**/*.ts"

